chat/src/main/java/com/qiscus/sdk/data/model/QiscusComment.java

```diff
@@ -88,6 +88,7 @@
 
     private String rawType;
     private String extraPayload;
+    private JSONObject extras;
```

Comment:
parcel bro masih ketinggalan @andhikayuana

Reply:
di writeToParcel() extrasnya belum di simpan, di constructor QiscusComment(Parcel in) jg belum ngambil data dari parcel

Reply:
okok :smile: parcel lebaran ketinggalan wkwk :v: